cli/src/klio_cli/cli.py

```diff
@@ -54,6 +55,7 @@
 DockerRuntimeConfig = collections.namedtuple(
     "DockerRuntimeConfig", ["image_tag", "force_build", "config_file_override"]
 )
+# TODO: We can get rid of direct_runner since we have runner
```

Comment:
This seems like a relevant comment for `RuntimeConfig` in `exec/src/klio_exec/cli.py` too.